google/cloud/bigquery_unified/CMakeLists.txt

```diff
@@ -178,11 +180,27 @@ bigquery_unified_library(
     PKGCONFIG_DEPS
     ${bigquery_unified_pkgconfig_deps})
 
+if (GOOGLE_CLOUD_CPP_BIGQUERY_ENABLE_OPENTELEMETRY)
+    find_package(google_cloud_cpp_opentelemetry CONFIG)
+    if (TARGET google-cloud-cpp::opentelemetry)
+        target_compile_definitions(
+            google_cloud_cpp_bigquery_bigquery_unified
+            PUBLIC # Enable OpenTelemetry features in google-cloud-cpp-bigquery
+                   GOOGLE_CLOUD_CPP_BIGQUERY_HAVE_OPENTELEMETRY)
+    else ()
+        message(
+            FATAL_ERROR
+                "OpenTelemetry is not enabled in the package "
+                "google-cloud-cpp, the target "
+                "google-cloud-cpp::opentelemetry is not available.")
```

Comment:
Ok, but what should the customer do?

"Instrumentation with OpenTelemetry was requested for `google-cloud-cpp-bigquery`, but it was not enabled in `google-cloud-cpp`. Please build and install `google-cloud-cpp` with `GOOGLE_CLOUD_CPP_ENABLE=opentelemetry,...` if you want to use this feature.

Reply:
Updated.